source/Calamari.Tests/Fixtures/Deployment/Conventions/AlreadyInstalledConventionFixture.cs

```diff
@@ -8,7 +8,7 @@
 using NSubstitute;
 using NUnit.Framework;
 
-namespace Calamari.Tests.Fixtures.Conventions
+namespace Calamari.Tests.Fixtures.Deployment.Conventions
```

Comment:
This namespace shuffle has added quite a lot of churn / noise to the PR. Perhaps we could separate this into two PRs: one that introduces the framework changes, and another that then does the "tidy up" of namespace reshuffling? It will make them easier to understand later on if people need to return to them.

Reply:
That sounds good to me